lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -172,19 +179,35 @@ data StakePoolLayer = StakePoolLayer
         -- Exclude all pools that retired in or before this epoch.
         -> Coin
         -> ExceptT ErrListPools IO [Api.ApiStakePool]
+
+    , putSettings :: Settings -> IO ()
+
+    , clearMetadataTable :: IO ()
+
+    , metadataThread :: TVar ThreadId
+
+    , metadataRun
+        :: IO ThreadId
```

Comment:
What's the difference between `run` and `thread` ?

Reply:
thread is the thread id, run runs the thread :man_shrugging: